### PR TITLE
Add de.k_bo.Televido

### PR DESCRIPTION
**Pull request by d-k-bo**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

---

The `--talk-name=*` permissions are used to start video streaming in the video player selected by the user (using the `org.freedesktop.Application.Open` method). The OpenURI portal can't be used since it would open the video URL in a web browser.

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
the upstream appstream file use a different app-id.

this will not build.

File: de.k_bo.Televido.json [hunk 1]
```diff
@@ -58,8 +58,8 @@
       "sources": [
         {
           "type": "archive",
-          "url" : "https://github.com/d-k-bo/televido/releases/download/v0.1.0/televido-0.1.0.tar.xz",
-          "sha256": "8abaf53646d278d69f78f985e33398f4e3e081b4f41f713ed8ec777bba38198c"
+          "url": "https://github.com/d-k-bo/televido/releases/download/v0.1.0/televido-0.1.0.tar.xz",
+          "sha256": "025df8987dd420e2d5d923fe13426bf9dfa7a90c67ae08e8838c5da415bca4a1"
```
Comment (hfiguiere) [inline de.k_bo.Televido.json:62]: retagging is bad.

Comment (d-k-bo) [reply de.k_bo.Televido.json:62]: Sorry, I'm pretty new to packaging. I believed this would be the right thing to do since the app itself didn't change.

Comment (d-k-bo) [reply de.k_bo.Televido.json:62]: It seems like I misunderstood the concept of git tags. I was not aware that they are meant to be immutable.

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
bot, build de.k_bo.Televido

**Comment by bbhtt** [comment]
bot, build de.k_bo.Televido

Builddirs probably has been purged I don't see any recent failed builds.

**Comment by d-k-bo** [comment]
I just tested the build and it looks fine. Thank you!

**Pull request closed without merging**